v8_c_api/src/v8_c_api.h

```diff
@@ -206,7 +206,7 @@ void v8_FreeContext(v8_context* ctx);
 
 /* Set a private data on the given context.
  * The private data can later be retrieve using `v8_GetPrivateData`. */
-void v8_SetPrivateData(v8_context* ctx, size_t index, void *pd);
+int v8_SetPrivateData(v8_context* ctx, size_t index, void *pd);
```

Comment:
Same comment as above, I do not see a need to return an indication if it succeeded or failed. We can assert that pd is not NULL and add an API to invalidate (somehow) the current pd.
WDYT?